app/controllers/account_price_group_members_controller.rb

```diff
@@ -1,63 +1,70 @@
 class AccountPriceGroupMembersController < ApplicationController
-  admin_tab     :all
-  before_filter :authenticate_user!
-  before_filter :check_acting_as
-  before_filter :init_current_facility
-
-  load_and_authorize_resource
-  
-  layout 'two_column'
-  
-  def initialize
-    @active_tab = 'admin_facility'
-    super
-  end
+  include PriceGroupMembersController
 
   # GET /price_group_members/new
   def new
-    @price_group = current_facility.price_groups.find(params[:price_group_id])
     @account_price_group_member = AccountPriceGroupMember.new
   end
 
   # GET /facilities/:facility_id/price_groups/:price_group_id/account_price_group_members/create
-  def create
-    @price_group = current_facility.price_groups.find(params[:price_group_id])
-    @account     = Account.find(params[:account_id])
-    @account_price_group_member = AccountPriceGroupMember.new(:price_group => @price_group, :account => @account)
-
-   if @account_price_group_member.save
-     flash[:notice] = "#{@account_price_group_member.account.account_number} was added to the #{@price_group.name} Price Group."
-   else
-     flash[:error] = "An error was encountered while trying to add account #{@account_price_group_member.account.account_number} to the #{@price_group.name} Price Group."
-   end
-   redirect_to([current_facility, @price_group])
-  end
-
-  # DELETE /price_group_members/1
-  def destroy
-    @price_group = current_facility.price_groups.find(params[:price_group_id])
-    @account_price_group_member = AccountPriceGroupMember.find(:first, :conditions => { :price_group_id => @price_group.id, :id =>params[:id]} )
-
-    if @account_price_group_member.destroy
-     flash[:notice] = "The account was successfully removed from the Price Group"
+  def create # TODO very similar to UserPriceGroupMembersController#create
+    if price_group_member.save
+      flash[:notice] = I18n.t("controllers.account_price_group_members.create.notice", create_flash_arguments)
```

Comment:
Could we use something like `controller_name` so we could share this code in the PriceGroupMembersController?